rpc_util.go

```diff
@@ -323,6 +324,25 @@ func (o MaxSendMsgSizeCallOption) before(c *callInfo) error {
 }
 func (o MaxSendMsgSizeCallOption) after(c *callInfo) {}
 
+// CallCompressionFactor returns a CallOption which sets the predicted compression factor,
+// used in sizing buffers
+func CallCompressionFactor(f int) CallOption {
```

Comment:
float32/64?  To allow more precise values like `1.5`?

(I'm also uncertain if the proposed default of 2x is acceptable, given that it means we are now allocating **3x** what the client sent, which could potentially be a vulnerability.  I'll ask some others for input.)